### DISCUSSION TITLE
Health checks  for USB HDD configured as internal drive.

### DISCUSSION THREAD

**Discussion by heikil** [007revad/Synology_HDD_db#74] [Ideas]
Hi!
Not sure this is the right place for a feature request, but here goes.
When config files are adjusted to include USB and eSATA drives as internal drives it is possible to use an eSATA drive to create a volume in a Synology NAS. Instructions are here https://xpenology.com/forum/topic/37503-today-i-learned/?tab=comments#comment-180605

This works fine with eSATA drives. However the USB drives report that health check is not available and thus it is not possible to create a new volume or use it as SSD cache. For the usb drive it is perfectly possible to run e.g. `sudo smartctl -a /dev/sdq` and view SMART data. Thus depending on USB - SATA bridge, the functionality is available.

If this is indeed possible or interesting for the dev(s) of this project, it would be a great feature to enable USB device health checks or maybe even adjusting config to treat them as internal drives.

PS. I am aware that some frown upon usb for any serious storage, but option to do so would be nice.

**Comment by 007revad** [comment]
Interesting.

I might add it to my [todo list](https://github.com/007revad/todo_list).
